src/main/java/core/basesyntax/model/Cat.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax.model;
+
+public class Cat {
+    private int age;
+    private String name;
+    private String ownerName;
+
+    @Override
+    public String toString() {
+        return "Cat{"
+                +
+                "age=" + age
+                +
+                ", name='" + name + '\''
+                +
+                ", ownerName='" + ownerName + '\''
+                +
+                '}';
```

Comment:
Please do not place `+` as the only character in a line. You did a good job in the `sayHello` method in the `HelloWorld` class, do it in a similar way.